chia/util/errors.py

```diff
@@ -186,7 +186,10 @@ class Err(Enum):
     # raised if a spend issues too many assert spend, assert puzzle,
     # assert announcement or create announcement
     TOO_MANY_ANNOUNCEMENTS = 144
-    CHIP_0013_VALIDATION = 145
+    # the specified coin ID is not valid
+    INVALID_COIN_ID = 145
+    # message not sent/received
+    MESSAGE_NOT_SENT_OR_RECEIVED = 146
```

Comment:
Would be cool if this could be two errors (since we have two opcodes even if that's somewhat superficial).  If the underlying implementation makes that difficult though, this is fine.

Reply:
the challenge is; if you send a message and you receive another message, you have an invalid spend bundle. But is it because the message you sent wasn't received or because the message you receive wasn't sent? It's both, so the error message you get depends on some implementation detail.

The cheap way of validating this won't let you make it deterministic (i.e. it would be more expensive to make it deterministic which failure you get). Having the failure not be deterministic makes testing harder. So, I went for a single message to mean "some message" failed to be sent or received.

Reply:
Yeah okay like I said, this is fine if it's too complicated I just know as a chialisp developer I am often going to be lamenting the lack of information.  Feel free to resolve.